lnd_grpc/lnd_grpc.py

```diff
@@ -64,15 +67,17 @@ def tls_cert_path(self, path):
     def tls_cert_key(self):
         try:
             with open(self.tls_cert_path, 'rb') as r:
-                self._tls_cert_key = r.read()
+                tls_cert_key = r.read()
```

Comment:
This variable was not being used anywhere else in the class, so I made it a local instead of a private attribute